src/components/FAIcon.js

```diff
@@ -8,6 +8,7 @@ const FAIcon = ({
   component: Component = 'i',
   extraClassName = '',
   modifiers = [],
+  noAriaHidden = false,
```

Comment:
I had actually done that on purpose with the same rationale behind `dangerouslySetInnerHTML` - icons should be default be `aria-hidden="true"` to avoid them being read out by screen readers and deviating from that behaviour requires special care. The `noAriaHidden` implies 'negativity' and draws attention for reviewers (I hope).

There is also the added benefit that you don't need the `true` or `false` for booleans that way, you could just use `<FAIcon icon="foo" noAriaHidden />` instead of the more verbose equivalent `<FAIcon icon="foo" ariaHidden={false} />`

Reply:
`<FAIcon icon="foo" noAriaHidden />` does look nicer: like in vim you can `:set hlsearsch` and `:set nohlsearch` to toggle.